topostats/plottingfuncs.py

```diff
@@ -203,6 +227,13 @@ def save_figure(self):
             )
             if isinstance(self.masked_array, np.ndarray):
                 self.masked_array[self.masked_array != 0] = 1
+                # If the image is too large for singles to be resolved in the mask, then dilate the mask proportionally
```

Comment:
I like this approach of only dilating if required and doing so automatically, means we don't have to add another option to the configuration file :+1: 

(Although I bet at some point someone will ask to configure the dilation!)